cypress/integration/userCanSelectActivities.feature.js

```diff
@@ -0,0 +1,17 @@
+describe("Activities", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/activities");
+  });
+  it("can see the destination chosen", () => {
+    cy.get(".activities").within(() => {
+      cy.get(".seven > :nth-child(2)").click();
```

Comment:
is it possible to make this more descriptive

Reply:
Made it more descriptive and put them in commands